evalml/tests/component_tests/test_estimators.py

```diff
@@ -25,7 +26,7 @@ def test_estimators_feature_name_with_random_ascii(X_y_binary, X_y_multi, X_y_re
             elif problem_type == ProblemTypes.REGRESSION:
                 X, y = X_y_regression
 
-            X = clf.random_state.random((X.shape[0], len(string.printable)))
+            X = get_random_state(clf.random_state).random((X.shape[0], len(string.printable)))
```

Comment:
Getting a little confused with why we're still using get_random_state.

Reply:
@chukarsten In some scenarios, we need to use RandomState.choice() to get a random choice of elements from lists. This primarily occurs with our use of an estimator with [random choice strategy](https://github.com/alteryx/evalml/blob/70c42b1943c3d9090cb5a8adea3731314bc48295/evalml/pipelines/components/estimators/classifiers/baseline_classifier.py#L68). In scenarios where we need to use random choice, I chose to keep the numpy RandomState. This PR primarily tracks to make sure our classes/objects don't use `np.random.RandomState` otherwise.

Reply:
Alright, that works!